code/drasil-example/glassbr/lib/Drasil/GlassBR/Changes.hs

```diff
@@ -30,7 +30,7 @@ accMoreThanSingleLite     = cic "accMoreThanSingleLite"     accMoreThanSingleLit
 accMoreBoundaryConditions = cic "accMoreBoundaryConditions" accMoreBoundaryConditionsDesc         "Accomodate-More-Boundary-Conditions" likeChgDom
 considerMoreThanFlexGlass = cic "considerMoreThanFlexGlass" considerMoreThanFlexGlassDesc         "Consider-More-than-Flexure-Glass"    likeChgDom
 
-calcInternalBlastRiskDesc :: NamedChunk -> Sentence
```

Comment:
And here too (see previous comment). I won't point them all out, you should re-review your changes to find these.